microsoft-365/security/office-365-security/how-atp-safe-links-works.md

```diff
@@ -63,4 +63,7 @@ At a high level, here's how [ATP Safe Links](atp-safe-links.md) protection works
       - If the URL is considered safe, the user is taken to the website.
       
       - If the URL check fails, Safe Links' protection will not trigger. On the desktop clients, the user will be warned before proceeding through to the site.
+      
+Note: It may take several seconds at the beginning of each session for Office to verify that the user has ATP Safe Links for Office enabled. 
```

Comment:
```suggestion
> [!NOTE]
> It may take several seconds at the beginning of each session for Office to verify that the user has ATP Safe Links for Office enabled. 
```

Reply:
I think "Office" is more accurate than "Office 365" here, because we're talking about the Office app boot-up.